cmd/create/cluster/cmd.go

```diff
@@ -1497,13 +1500,15 @@ func run(cmd *cobra.Command, _ []string) {
 			if isHostedCP {
 				createAccountRolesCommand = createAccountRolesCommand + " " + hostedCPFlag
 			}
-			r.Reporter.Warnf(fmt.Sprintf("No compatible account roles with version '%s' found. "+
-				"You will need to manually set them in the next steps or run '%s' to create them first.",
-				minor, createAccountRolesCommand))
+			r.Reporter.Warnf("No suitable accounts with ROSA CLI-created account roles were found. "+
+				"You can manually set them in the next steps or run '%s' to create them first.", createAccountRolesCommand)
```

Comment:
I believe it is only looking in a single account here, rather than multiple, please LMK if that assumption is incorrect

But if one account I think we can remove `accounts with` here

Reply:
should be fixed here [6a52f12](https://github.com/openshift/rosa/pull/2867/commits/6a52f123604cf00d48743681600fb9af9538cf64)